src/Raven.Client/Documents/Operations/Revisions/ConfigureRevisionsOperation.cs

```diff
@@ -13,24 +13,28 @@ namespace Raven.Client.Documents.Operations.Revisions
     public class ConfigureRevisionsOperation : IMaintenanceOperation<ConfigureRevisionsOperationResult>
     {
         private readonly RevisionsConfiguration _configuration;
+        private readonly bool _enforce;
 
-        public ConfigureRevisionsOperation(RevisionsConfiguration configuration)
+        public ConfigureRevisionsOperation(RevisionsConfiguration configuration, bool enforce = false)
```

Comment:
This breaks binary compatibility. You need a separate 'ctor'.

Reply:
done